src/server/routes/configuration/ip/set.ts

```diff
@@ -0,0 +1,63 @@
+import { Static, Type } from "@sinclair/typebox";
+import { FastifyInstance } from "fastify";
+import { StatusCodes } from "http-status-codes";
+import { updateConfiguration } from "../../../../db/configuration/updateConfiguration";
+import { getConfig } from "../../../../utils/cache/getConfig";
+import { standardResponseSchema } from "../../../schemas/sharedApiSchemas";
+import { responseBodySchema } from "./get";
+
+const requestBodySchema = Type.Object({
+  ips: Type.Array(
+    Type.String({
+      minLength: 7,
+      description: "IP Address as a string",
+    }),
+    {
+      description: "Array of IP addresses to allowlist",
+    },
+  ),
+});
+
+requestBodySchema.examples = [
+  {
+    ips: ["8.8.8.8", "172.217.255.255"],
+  },
+];
+
+export async function setIpAllowlist(fastify: FastifyInstance) {
+  fastify.route<{
+    Body: Static<typeof requestBodySchema>;
+    Reply: Static<typeof responseBodySchema>;
+  }>({
+    method: "PUT",
+    url: "/configuration/ip-allowlist",
+    schema: {
+      summary: "Set IP Allowlist",
+      description:
```

Comment:
Remove the `client-side` part. IP allowlist is useful for restricting API calls from servers too.